tests/__init__.py

```diff
@@ -157,8 +158,9 @@ def run_server(server_urls=None, port=None):
     with TemporaryDirectory() as tmp_dir_name:
         logger.info('created temporary directory %s' % tmp_dir_name)
         config_file_path = tmp_dir_name + '/config.ini'
-        config_data = '[DEFAULT]\nDIRECTORY = %s\nLOG_LEVEL = INFO\nPORT = %d\nTesting = True\n"BOUNDING_BOX_MINIMUM_DP = 2\nBOUNDING_BOX_MAXIMUM_SIZE = 0.001\nLOCATION_RESOLUTION = 4\nAPP_TESTING = 2.0\n' % (
-            tmp_dir_name, port)
+        log_file_path = tmp_dir_name + '/log.txt'
```

Comment:
If this is a log_file_path why are we calling it log_file_name - that's confusing (I presume name is "log.txt" path is "/tmp/xxx/log.txt"

Reply:
i'll fix